x-pack/plugins/security_solution/public/common/components/response_actions/endpoint_action_results.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiComment, EuiLoadingSpinner, EuiSpacer } from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n-react';
+import React, { useEffect, useState, useMemo } from 'react';
+import type { LogsEndpointActionWithHosts } from '../../../../common/endpoint/types/actions';
+import { useUserPrivileges } from '../user_privileges';
+import { useGetAutomatedActionResponseList } from '../../../management/hooks/response_actions/use_get_automated_action_list';
+import { ActionsLogExpandedTray } from '../../../management/components/endpoint_response_actions_list/components/action_log_expanded_tray';
+import type { ResponseActionsApiCommandNames } from '../../../../common/endpoint/service/response_actions/constants';
+import { ENDPOINT_COMMANDS } from '../event_details/translations';
+import { ResponseActionsEmptyPrompt } from './response_actions_empty_prompt';
+
+interface EndpointResponseActionResultsProps {
+  action: LogsEndpointActionWithHosts;
+}
+
+export const EndpointResponseActionResults = ({ action }: EndpointResponseActionResultsProps) => {
+  const { rule, agent } = action;
+  const { action_id: actionId, expiration } = action.EndpointActions;
+  const {
+    endpointPrivileges: { canReadActionsLogManagement, canAccessEndpointActionsLogManagement },
+  } = useUserPrivileges();
+
+  const [isLive, setIsLive] = useState(true);
+  const canReadEndpoint = canReadActionsLogManagement && canAccessEndpointActionsLogManagement;
```

Comment:
Sorry, I missed this earlier as well. This condition boils down to having READ RBAC access to Actions Log management and having platinum and enterprise licenses. So this won't work if you're not `superuser`.

We should decide if we want to give this to enterprise or platinum users. One can't have two licenses at the same time. For platinum and upward only `canReadActionsLogManagement` will do, and for `enterprise` only `canAccessEndpointActionsLogManagement` should be sufficient.

Reply:
Yeah, this is also okay. 

But I understand that, if you gate this for `platinum` users then it will work for both platinum and enterprise users. Enterprise includes everything platinum has. So in essence, just `canReadActionsLogManagement` will do.

You can also add a test for these license checks if you like.